cli/src/cli.ts

```diff
@@ -94,8 +94,8 @@ const config: Config = {
     useGet: program.get,
     schema: program.schema && readFile(program.schema),
     output: program.output,
-    headers: parseColonSeparatedStrings(program.header),
-    scalarTypes: parseColonSeparatedStrings(program.scalar),
+    headers: parseColonSeparatedStrings(program.header || []),
```

Comment:
did you try if this is needed? the function header of parseColonSeparatedStrings is:

```typescript
function parseColonSeparatedStrings(array: string[] = [])
```
that transpiles to 

```js
function parseColonSeparatedStrings(array) {
    if (array === void 0) { array = []; }
```
so... as I see it, you created duplicated code @remorses - or am I overseeing sth?

Reply:
You are probably right, the || just makes sure to handle null or “” values, I just want to be safe 😀